scripts/create-kubestellar-demo-env.sh

```diff
@@ -139,9 +104,83 @@ checking_cluster() {
 
     done
 }
+##########################################
+
+echo -e "\nStarting environment clean up..."
+echo -e "Starting cluster clean up..."
+
+cluster_clean_up "kind delete cluster --name kubeflex"
+cluster_clean_up "kind delete cluster --name cluster1"
+cluster_clean_up "kind delete cluster --name cluster2"
+echo -e "Cluster space clean up has been completed"
+
+echo -e "\nStarting context clean up..."
+
+context_clean_up
+echo "Context space clean up completed"
+
+echo -e "\nStarting the process to install KubeStellar core: kind-kubeflex..."
+
+clusters=(cluster1 cluster2)
+
+for cluster in "${clusters[@]}"; do
+   (
+     echo -e "Creating cluster ${cluster}..."
+     kind create cluster --name "${cluster}" >/dev/null 2>&1 &&
+     kubectl config rename-context "kind-${cluster}" "${cluster}" >/dev/null 2>&1
+     echo -e "${cluster} creation and context setup complete"
+   ) &
+done
+
+wait 
+
+echo -e "Creating KubeFlex cluster with SSL Passthrough"
+curl -s https://raw.githubusercontent.com/kubestellar/kubestellar/v${KUBESTELLAR_VERSION}/scripts/create-kind-cluster-with-SSL-passthrough.sh | bash -s -- --name kubeflex --port 9443 
```

Comment:
The capitalization of the environment variable name needs to be fixed here and in the later places where it is still uppercase.

Reply:
fixed